src/main/java/com/infogain/debonair/repository/Menu.java

```diff
@@ -0,0 +1,62 @@
+package com.infogain.debonair.repository;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.stereotype.Repository;
+
+import com.infogain.debonair.dto.Items;
+
+/**
+ * Menu class is a repository which handles Menu operations
+ * 
+ * @author Chetna
+ * @since 21-Apr-2020
+ */
+@Repository
+public class Menu {
+	private List<Items> ITEM_MENU = new ArrayList<>();
+
+	public List<Items> getITEM_MENU() {
+		return ITEM_MENU;
+	}
+
+	public void setITEM_MENU(List<Items> iTEM_MENU) {
+		ITEM_MENU = iTEM_MENU;
+	}
+
+	/**
+	 * This method is used to fetch menu from Items.txt file
+	 * 
+	 * @return
+	 * @since 04-May-2020
+	 */
+	public List<Items> fetchMenu() {
+		// fetching items from Items.txt file and saving it in itemMenu
+		// collection(ArrayList)
+		ITEM_MENU.clear();
+		try (BufferedReader bufferedReaderObj = new BufferedReader(new FileReader("./src/main/resources/Items.txt"))) {
```

Comment:
This method will always read from file what is the use of the private List<Items> ITEM_MENU = new ArrayList<>();???